git-secrets-scan-action/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This should be Apache 2.0 and attributed to Amazon (if applicable to license text).

Reply:
Done. Seems like aws-s3-cp action is still having a MIT license (will make that change in a new PR)

Reply:
By the way, the license for `s3-cp-action` doesn't need to change because that's effectively a fork of https://github.com/jakejarvis/s3-sync-action.